README.md

```diff
@@ -2,7 +2,7 @@
 Ansible automation to load test-data of various sample applications to aid OCP Migration development efforts.
 
 # Usage
-For backups/restores ark is required, it can be install using playbooks from this repo ```https://github.com/fusor/ocp-velero-ansible```
+Ark is required for backups/restores, it can be installed using playbooks from this repo ```https://github.com/fusor/ocp-velero-ansible```
```

Comment:
I'd suggest to use the term 'Velero' on anything new

Reply:
Sure, the new name just doesn't stick :)